README.md

```diff
@@ -16,7 +16,7 @@ Check out the [video tutorial](https://drive.google.com/file/d/1RNLQYlgJqE8JMv0n
 
 ## Releases
 
-Latest release: [DeepOps 21.03 Release](https://github.com/NVIDIA/deepops/releases/tag/21.03)
+Latest release: [DeepOps 21.05 Release](https://github.com/NVIDIA/deepops/releases/tag/21.05)
```

Comment:
Can you make this 21.06?

Reply:
Damn, I had one job! sorry, forgot to commit